source/scss/_pf-component-view-selector.scss

```diff
@@ -0,0 +1,17 @@
+//
+// View Selector component
+//
+
+.pf-view-selector__btn-link {
```

Comment:
Ok, let's keep a good eye on PR reviews and be very clear on documenting this. If not we'll end up with a mess.